src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -51,6 +51,12 @@ public class AnalyzeOptionsBase : IAnalyzeOptions
         public bool ComputeTargetsHash { get; set; }
 
         [Option(
+            'e',
+            "environment",
+            HelpText = "Log machine environment details of run to output file. WARNING: This option records potentially sensitive information (such as all environment variable values) to any emitted log.")]
+        public bool LogEnvironment { get; set; }
+
```

Comment:
Oh, I see. I finally got down to the implementation of `Invocation.Create`. It doesn't mean "environment variables", it means "the whole execution environment". Ok.